specs/public_inputs.md

```diff
@@ -0,0 +1,263 @@
+# Public Inputs
+
+The following list contains all the data used by an Ethereum client to
+calculate a block and to define this calculated block.
+
+## Block
+
+- Hash: 256 bits
+- Block Fields used in RLP:
+    - ParentHash: 256 bits
+    - UncleHash: 256 bits
+    - Coinbase: 160 bits
+    - Root: 256 bits (State Trie Root)
+    - TxHash: 256 bits (Txs Trie Root)
+    - ReceiptHash: 256 bits (Receipts Trie Root)
+    - Bloom: 256 bytes
+    - Difficulty: 256 bits
+    - Number: 64 bits
+    - GasLimit: 64 bits
+    - GasUsed: 64 bits
+    - Time: 64 bits
+    - Extra: 0 bytes
+    - MixDigest: 256 bits
+    - Nonce: 64 bits
+    - BaseFee: 256 bits (BaseFee was added by EIP-1559 and is ignored in legacy headers.)
+
+### Circuits
+
+- Block Hash verifier
+    - All fields
+- EVM Circuit Block Table
+    - Coinbase
```

Comment:
I see `ChainId` put into block scope fields, Would you like to add it here ?

Reply:
The `ChainID` is technically not a block field, although we add it in the block table for lookup convenience.  This fact is reflected already here https://github.com/privacy-scaling-explorations/zkevm-specs/blob/a3ab4a09d4ae4d832fb7abe06f381454f74302b1/specs/public_inputs.md?plain=1#L53-L60